README.md

```diff
@@ -127,6 +127,36 @@ Please note for this to work you should create/update `${GRADLE_USER_HOME}/gradl
     signing.password=
     signing.secretKeyRingFile=
 
+### Publishing the streams quickstart archetype artifact to maven ###
+For the Streams archetype project, one cannot use gradle to upload to maven; instead the `mvn deploy` command needs to be called at the quickstart folder:
+
+    cd streams/quickstart
+    mvn deploy
+
+Please note for this to work you should create/update either user maven settings (typically, `${USER_HOME}/.m2/settings.xml`) or global maven settings (typically, `${MAVEN_HOME}/conf/settings.xml`) to assign the following variables
```

Comment:
generally you shouldn't ever modify that global config unless you're working on some shared environment where that would make sense. just the `~/.m2` part should be enough